osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -49,13 +49,23 @@ public partial class BeatmapCarousel : CompositeDrawable, IKeyBindingHandler<Glo
         /// </summary>
         public Action? BeatmapSetsChanged;
 
+        /// <summary>
+        /// Triggered after filter conditions have finished being applied to the model hierarchy.
+        /// </summary>
+        public Action? FilterApplied;
+
         /// <summary>
         /// The currently selected beatmap.
         /// </summary>
         public BeatmapInfo? SelectedBeatmapInfo => selectedBeatmap?.BeatmapInfo;
 
         private CarouselBeatmap? selectedBeatmap => selectedBeatmapSet?.Beatmaps.FirstOrDefault(s => s.State.Value == CarouselItemState.Selected);
 
+        /// <summary>
+        /// The total count of non-filtered beatmaps displayed.
+        /// </summary>
+        public int CountDisplayed => beatmapSets.Where(s => !s.Filtered.Value).Sum(s => s.Beatmaps.Count(b => !b.Filtered.Value));
```

Comment:
I'd like to see this done as part of the filter process, rather than a post-execution linq. But I don't want to refactor the filter process too much just for this change.